x-pack/plugins/observability/server/lib/rules/custom_threshold/translations.ts

```diff
@@ -22,6 +22,30 @@ export const AVERAGE_I18N = (metric: string) =>
     },
   });
 
+export const PERCENTILE_99_I18N = (metric: string) =>
+  i18n.translate('xpack.observability.customThreshold.rule.aggregators.p99', {
+    defaultMessage: '99th Percentile {metric}',
```

Comment:
```suggestion
    defaultMessage: '99th percentile {metric}',
```

percentile with lowercase `p` seems better to me, @paulb-elastic @vinaychandrasekhar what do you think?

Currently, it is like this:

> 95th Percentile system.cpu.total.norm.pct is 90%, above the threshold of 80%. (duration: 1 min, data view: metrics-fake_hosts)

Reply:
I think those are good suggestions